week_1/__tests__/String.test.js

```diff
@@ -0,0 +1,65 @@
+const isAnagram = require("../String/valid_anagram");
+const isPalindrome = require("../String/valid_palindrome");
+const romanToInt = require("../String/roman_to_int");
+const {
+	longest_palindrome,
+	longestPalindrome,
+} = require("../String/longest_palindrome");
+const isValidParentheses = require("../String/valid_parentheses");
+const isSubsequence = require("../String/is_subsequence");
+const longestCommonPrefix = require("../String/longest_common_prefix");
+
+test("valid_anagram works", () => {
```

Comment:
All these tests are great, but let's keep this MR scoped to something manageable: the prefix algorithm. We can split this diff off into a separate MR and get that merged independently :)